_pages/platform/account-profile.md

```diff
@@ -0,0 +1,103 @@
+
+### Errors
+Beginning in Algorithmia version 20.5.52, this tab displays runtime errors from algorithms that you've called. Specifically, it lists the 200 most recent algorithm runtime errors over the past seven days that've been called using any of your API keys.
+
+For each error, the algorithm endpoint is listed along with the exact date and time that the error occurred. If the error is a [custom Algorithmia error type]({{site.baseurl}}/algorithm-development/algorithm-errors), that information is displayed as well. The error message is displayed in red below these details.
+
+To expose additional details about the error and associated execution metadata, click the drop-down arrow to the right. The request ID, the worker node that handled the request, the first 128 characters of the input to the algorithm, and the expanded stack trace information are displayed.
+
+![Viewing algorithm runtime errors]({{site.cdnurl}}{{site.baseurl}}/images/post_images/your_profile/account-profile-errors.png)
+
+### Account
+
+This tab is only visible if you're **not** using the Algorithmia Enterprise platform.
+{: .notice-info}
+
+This tab displays information about your account usage and your credits.
+
+![Credits main view]({{site.cdnurl}}{{site.baseurl}}/images/post_images/your_profile/credits_main.png)
+
+The card on the left shows details about your account, such as the number of free credits remaining in the current 30-day cycle as well as the total number of credits available, including your purchased credits. You can also purchase more credits through this page by clicking the **Purchase Credits** button. The card to the right displays your payment information, if any, and allows you to update or remove your current payment method.
+
+If you've earned credits by publishing your own algorithms, you'll see the amount you've earned (in USD) as well.
+
+Below the account overview, you'll see a chart of your balance over time, then menu options to view your algorithm usage, your earnings, and your credit transactions.
+
+#### Usage
+
+This section displays the algorithms you've called, the date of each call, and associated usage statistics.
+
+The **number of calls** column lists the number of times that you've called each algorithm, but you'll see that sometimes the **billed calls** column displays a lower value. This is because we don't bill API calls that result in an error.
+{: .notice-info}
+
+The **billed time** column lists the time billed, in seconds, which is the time it took for the algorithm to execute on the platform. We use this compute time and the royalty on the algorithm to calculate the value in the **cost in credits** column.
+![Algorithm usage]({{site.cdnurl}}{{site.baseurl}}/images/post_images/your_profile/usage.png)
+
+For more information about pricing, see [Pricing]({{site.baseurl}}/pricing).
+
+#### Earnings
```

Comment:
I don't think this exists on the user page.

Reply:
which part doesn't exist?

![image](https://user-images.githubusercontent.com/36384768/121570190-8070ea00-c9d6-11eb-9bcf-53b5af932f60.png)

Reply:
ah tabs under tabs. I get it now.